vignettes/s3-vector.Rmd

```diff
@@ -643,18 +641,18 @@ These proxy functions should always return a simple object (typically either a b
 
 ### Rational class
 
-Let's explore these ideas by with the rational class we started on above. By default, `vec_proxy_equal()` converts a record to a data frame, and the default comparison works column by column:
+Let's explore these ideas by with the rational class we started on above. By default, `vec_proxy()` converts a record to a data frame, and the default comparison works column by column:
 
 ```{r}
 x <- rational(c(1, 2, 1, 2), c(1, 1, 2, 2))
 x
 
-vec_proxy_equal(x)
+vec_proxy(x)
 
 x == rational(1, 1)
 ```
 
-This makes sense as a default, but isn't correct here because `rational(1, 1)` represents the same number as `rational(2, 2)` so they should be equal. We can fix that by implementation `vec_proxy_equal()` method that by divides `n` and `d` by their greatest common divisor:
+This makes sense as a default, but isn't correct here because `rational(1, 1)` represents the same number as `rational(2, 2)` so they should be equal. We can fix that by implementation `vec_proxy()` method that by divides `n` and `d` by their greatest common divisor:
```

Comment:
```suggestion
This makes sense as a default, but isn't correct here because `rational(1, 1)` represents the same number as `rational(2, 2)` so they should be equal. We can fix that by implementing a `vec_proxy()` method that by divides `n` and `d` by their greatest common divisor:
```